cmd/dashboard.go

```diff
@@ -165,7 +165,7 @@ dapr dashboard -k -p 0
 			}()
 
 			// url for dashboard after port forwarding.
-			var webURL string = net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort))
+			var webURL string = fmt.Sprintf("%s%s", "http://", net.JoinHostPort(dashboardHost, fmt.Sprint(portForward.LocalPort)))
```

Comment:
dashboardHost only accepts IP or `localhost` . That is already validated before in a `utils.IsAddressLegal` call